### PR TITLE
Fix: thread race in bmqimp_brokersession.t.cpp

### PR DESCRIPTION
**Pull request by 678098**
* Fixed incorrect check for empty write calls after async Configure (must be non-empty after a small timeout)
* Fixed incorrect absolute timestamp for timedPopFront when waiting for events

### CODE REVIEW

File: src/groups/bmq/bmqimp/bmqimp_brokersession.t.cpp [hunk 1]
```diff
@@ -2242,7 +2242,6 @@ bool TestSession::waitForQueueRemoved(
 
     const bsls::TimeInterval expireAfter =
         bsls::SystemTime::nowRealtimeClock() + timeout;
-    mwcsys::Time::nowMonotonicClock() + timeout;
```
Comment (678098) [inline src/groups/bmq/bmqimp/bmqimp_brokersession.t.cpp:2245]: Unused statement

File: src/groups/bmq/bmqimp/bmqimp_brokersession.t.cpp [hunk 2]
```diff
@@ -9812,10 +9811,26 @@ static void test54_distributedTrace()
...
             ASSERT_NE(events.timedPopFront(&emptyStr,
-                                           mwcsys::Time::nowMonotonicClock() +
+                                           bdlt::CurrentTime::now() +
                                                bsls::TimeInterval(0.01)),
                       0);
         }
+
+        
+        static bool waitForWriteCall(const bsl::deque<mwcio::TestChannel::WriteCall> &calls,
+                                     const bsls::TimeInterval& timeout)
```
Comment (678098) [inline src/groups/bmq/bmqimp/bmqimp_brokersession.t.cpp:9821]: Similar to some other `waitFor` functions at the top of the file, but the scope is local to test54.

**Comment by 678098** [comment]
@sgalichkin 
`ctest -R bmqimp_brokersession`

**Comment by 678098** [comment]
Branch for this PR is in the main repository, It's impossible for me to force-push fixed commits with signed-off-by message here, here is another PR:
https://github.com/bloomberg/blazingmq/pull/59

**Comment by sgalichkin** [comment]
Ok!
________________________________

**Pull request closed without merging**